server/test/services/applicant/ApplicantServiceTest.java

```diff
@@ -2021,6 +2021,74 @@ public void relevantProgramsForApplicant_hiddenFromIndex() {
         .containsExactly(programDefinition.id());
   }
 
+  @Test
+  public void relevantProgramsForApplicant_disabled() {
```

Comment:
**suggestion:** It looks like this is currently testing two cases - when a program with a draft application gets disabled and when a program with a submitted application gets disabled. Can we split these into 2 separate tests?

Reply:
done